standardtracer/propagation.go

```diff
@@ -127,7 +133,7 @@ func (p *splitBinaryPropagator) InjectSpan(
 	}
 
 	// Handle the trace and span ids, and sampled status.
-	contextBuf := new(bytes.Buffer)
+	contextBuf := bytes.NewBuffer(splitBinaryCarrier.TracerState[:0])
```

Comment:
for my information: the parameter is just to take a guess at the size/capacity, right?